yassersharabati/main.py

```diff
@@ -0,0 +1,50 @@
+from fastapi import FastAPI, HTTPException
+from uuid import uuid4
+from typing import List
+from task import *
+
+app = FastAPI()
+
+
+tasks = []
```

Comment:
I think it is better to use `dict/object` instead of `list` for the tasks.
When you use it, you will have the ability to get any task by its id in O(1) with object reference. Now please edit it and it dependencies